### DISCUSSION TITLE
華碩、中山附醫合推門診 App，運用 AI 推薦看診科別、示警慢性病風險

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS3073-C/Python-Community-GS3073-C#85] [General]
**華碩 AI 研發中心（AICS）與中山醫學大學附設醫院推出《中山附醫 e 點通》App，提供 AI 推薦看診科別與醫生、線上門診報到、即時查詢看診與抽血叫號進度等，診後不僅能線上預約慢箋取藥回診，也能即時接收 AI 警示慢性病罹患風險與處置建議，以 AI 與行動服務提升患者體驗。未來這項服務將整合遠距醫療照護，實現個人化全方位健康管理。**
全文內容:https://technews.tw/2021/09/24/asus-aics-and-chung-shan-medical-university-hospital/
這功能我看是非常方便的，雖然我還沒有慢性病但至少去醫院不會迷路。

**Comment by Timmy110403540** [comment]
跟我上一篇文章似乎有些相似點!! 都運用AI到醫療上!
